tsdown.config.ts

```diff
@@ -1,10 +1,10 @@
-import { defineConfig } from "tsup";
+import { defineConfig } from "tsdown";
 
 export default defineConfig({
-	bundle: false,
 	clean: true,
 	dts: true,
 	entry: ["src/**/*.ts", "!src/**/*.test.*"],
 	format: "esm",
```

Comment:
FYI: The format is `es` (equal to `esm`) [by default](https://github.com/rolldown/tsdown/blob/d8edf7729cabd4748bc1b7d65f9db6e3bbec50a0/src/options/types.ts#L104). Still might be worth to keep it as explicit option